fitbit-react/src/Pages/Cart/HorizontalCard/index.js

```diff
@@ -2,5 +2,13 @@ import { AiFillMinusCircle } from "react-icons/ai";
 import { IoIosAddCircle } from "react-icons/io";
 import { PrimaryButton } from "../../../Components/Navbar";
 import { SecondaryButton } from "../../../Components/VerticalCard";
```

Comment:
changes: I think there is no need to export components or context or other folder items from here because you already export those files of respective folders from that folder itself.
like all components folder components will be exported from components/index.js.